src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_TicketingStack/TicketingTile.tsx

```diff
@@ -35,7 +35,7 @@ export const TicketingTile = ({
   title?: string;
   description?: string;
   accessibilityLabel?: string;
-  showBetaTag?: boolean;
+  showInfoTag?: 'new' | 'beta';
```

Comment:
`showInfoTag` can be renamed to just `infoTag` or `infoTagMode`, as the value is a string for the InfoTag mode, not a boolean.